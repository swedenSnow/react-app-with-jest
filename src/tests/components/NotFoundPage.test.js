import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../components/NotfoundPage';

describe('<NotFoundPage/>', () => {
    it('should render the 404 correctly', () => {
        const wrapper = shallow(<NotFoundPage />);
        console.log(wrapper.debug());
        expect(wrapper).toMatchSnapshot();
    });
    it('should have a set amount of children', () => {
        const wrapper = shallow(<NotFoundPage />);
        expect(wrapper.children()).toHaveLength(2);
    });
});

// test('should render NotFoundPage correct', () => {
//   const wrapper = shallow(<NotFoundPage />);
// });
