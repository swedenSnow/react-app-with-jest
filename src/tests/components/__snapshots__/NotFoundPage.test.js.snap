// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage/> should render the 404 correctly 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
